examples/beta/azure-container-apps-arm/README.md

```diff
@@ -0,0 +1,127 @@
+# Deploy 1Password Connect Server on Azure Container Apps using the Azure Portal
+
+_Learn how to deploy 1Password Connect Server on the [Azure Container Apps](https://azure.microsoft.com/en-us/products/container-apps/#overview) service._
+
+This deployment consists of two [containers](https://learn.microsoft.com/en-us/azure/container-apps/containers): One for the Connect API and another for Connect Sync. There's also an [ingress](https://learn.microsoft.com/en-us/azure/container-apps/ingress-overview) for the API container. There are a few benefits to deploying 1Password Connect Server on Azure Container Apps:
+
+- **Low cost:** For standard deployments, the service will host your Connect Server for ~$16 USD/month (as of January 2024). Container Apps pricing is variable based on activity, and you can learn more on [Microsoft's pricing page](https://azure.microsoft.com/en-us/pricing/details/container-apps/).
+- **Automatic DNS record management:** You don't need to manage a DNS record. Azure Container Apps automatically provides a unique one for your Connect Server domain.
+- **Automatic TLS certificate management:** Azure Container Apps automatically handles TLS certificate management on your behalf.
+
+## Before you begin
+
+Before you begin, ensure you have an Azure account with permission to create a Container App. You'll also need the `1password-credentials.json` file for your 1Password Connect Server, which contains the credentials for authenticating the server.
```

Comment:
Should we point to [the docs](https://developer.1password.com/docs/connect/get-started#step-1) around how to create this credentials file?